pkg/osv/osv.go

```diff
@@ -16,7 +16,7 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-const (
+var (
```

Comment:
Is this change intentional?

Reply:
I changed these from const to var so previously I could adjust the endpoint, I will revert back to const